test/api_integration/apis/saved_objects_management/relationships.ts

```diff
@@ -64,7 +73,7 @@ export default function ({ getService }: FtrProviderContext) {
           .get(relationshipsUrl('search', '960372e0-3224-11e8-a572-ffca06da1357'))
           .expect(200);
 
-        expect(resp.body).to.eql([
+        expect(resp.body.relations).to.eql([
```

Comment:
Note: the `'/api/kibana/management/saved_objects/relationships/{type}/{id}'` endpoint response is modified by this PR. it was directly returning the list of relation as root payload, and is now an object with `relations` and `invalidRelations` keys. I think this is alright to introduce this in a minor though, as this API is undocumented and 'internal', even if on the `/api` prefix. 

wdyt?

Reply:
yeah it's internal so we can make breaking changes here :+1: I also don't think we should create create release notes for internal breaking changes, it will just create noise for API's we anyway don't want users to consume.